src/ClusterBootstrap/params.py

```diff
@@ -664,14 +664,15 @@
         "runscriptonroles infra worker ./scripts/prepare_vm_disk.sh",
         "nfs-server create",
         "runscriptonroles infra worker ./scripts/prepare_ubuntu.sh",
+        "runscriptonroles infra worker ./scripts/disable_kernel_auto_updates.sh",
+        "runscriptonroles infra worker ./scripts/docker_network_gc_setup.sh",
         "genscripts",
         "runscriptonroles infra worker ./scripts/dns.sh",
         "-y deploy",
         "-y updateworker",
         "-y kubernetes labels",
         "-y gpulabel",
         "kubernetes start nvidia-device-plugin",
-        "kubernetes start flexvolume",
```

Comment:
Why remove this?

Reply:
I didn't remember removing it...maybe it's not synced. I don't know when is this added to the pipeline.